### PR TITLE
Update dependencies and replace PSR-2 by PSR-12

### PR DESCRIPTION
**Pull request by Vagnerlg**
This PR change the code standard's ruleset in order to adopt PSR12.
Also, this PR updates Slevomat and squizlabs dependencies

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: '2'
+services:
+  php:
+    image: 'leroymerlinbr/php:7.3-apache'
```
Comment (ravanscafi) [inline docker-compose.yml]: No need to use apache here. Also, I think that we can move on to php 7.4

```suggestion
    image: 'leroymerlinbr/php:7.4'
```

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    volumes:
+      - .:/var/www/html
```
Comment (ravanscafi) [inline docker-compose.yml]: can you add an empty line at the end?

File: lib/LeroyMerlin/ruleset.xml [hunk 3]
```diff
@@ -6,16 +6,11 @@
...
     <description>The coding standard for PHP projects on LMBR</description>
 
-    <!-- Import PSR-2 coding standard -->
-    <rule ref="PSR2">
-        <!-- Checked by SlevomatCodingStandard.Namespaces.UseSpacing -->
-        <exclude name="PSR2.Namespaces.UseDeclaration.SpaceAfterLastUse"/>
+    <rule ref="PSR12">
         <!-- Checked by SlevomatCodingStandard.Namespaces.NamespaceSpacing -->
-        <exclude name="PSR2.Namespaces.NamespaceDeclaration.BlankLineAfter"/>
+        <exclude name="PSR12.Files.FileHeader.SpacingAfterBlock"/>
```
Comment (ravanscafi) [inline lib/LeroyMerlin/ruleset.xml]: I think that we should use PSR12 completely and stop ignoring it.

File: lib/LeroyMerlin/ruleset.xml [hunk 4]
```diff
@@ -285,6 +272,8 @@
         <exclude name="Squiz.Commenting.FunctionComment.ThrowsNoFullStop"/>
         <!-- Doesn't work with self as typehint -->
         <exclude name="Squiz.Commenting.FunctionComment.TypeHintMissing"/>
+        <!-- We don't use column identation on doc blocks -->
+        <exclude name="Squiz.Commenting.FunctionComment.SpacingAfterParamType"/>
```
Comment (ravanscafi) [inline lib/LeroyMerlin/ruleset.xml]: what does this does? I do think we use it :thinking:

**Review by ravanscafi — APPROVED** [review]
Great job!

**Review by joaomarcusjesus — APPROVED** [review]

**Review by jefflssantos — APPROVED** [review]

File: tests/expected/example-class.php [hunk 5]
```diff
@@ -18,27 +22,25 @@ class Example implements IteratorAggregate
...
      * @var ControlStructureSniff|int|string|null
      */
     private $baxBax;
 
-    public function __construct(int $foo = null, array $bar = [], bool $baz = false, $baxBax = 'unused')
+    /**
+     * @param ControlStructureSniff|int|string|null $baxBax
+     * @param string[]                              $bar
+     */
+    public function __construct(?int $foo = null, array $bar = [], bool $baz = false, $baxBax = 'unused')
```
Comment (dhalsimbr) [inline tests/expected/example-class.php:43]: Não seria melhor a quebra de linha nos parâmetros?

File: tests/input/LowCaseTypes.php [hunk 6]
```diff
@@ -1,5 +1,4 @@
 <?php
-
```
Comment (dhalsimbr) [inline tests/input/LowCaseTypes.php:2]: Estas classes com namespace Test (LowCaseTypes.php, forbidden-comments.php, forbidden-functions.php, namespaces-spacing.php), estão sem espaço, mas em outras classes o espaço antes foi adicionado.

**Review by dhalsimbr — APPROVED** [review]

File: tests/expected/example-class.php (hunk 5 above)
Comment (gabrielgomes94) [reply tests/expected/example-class.php:43]: seria. vai entrar uma regra pra isso ao longo dos proximos PRs(quebrar parametros se passar de 80 caracteres na linha)

**Pull request merged**